velox/docs/functions/spark/non_deterministic_function.rst

```diff
@@ -0,0 +1,6 @@
+====================================
+Non Deterministic Functions
+====================================
```

Comment:
Why do we regard it as a non-deterministic function? It seems the function `isDeterministic()` is not overriden.
https://github.com/facebookincubator/velox/blob/main/velox/docs/develop/scalar-functions.rst#input-vectors